frontend/pages/app/league-info.vue

```diff
@@ -0,0 +1,198 @@
+
+
+<template>
+  <div>
+
+    <AppHeader :name="Home"/>
+
+    <!-- League Info -->
+    <div class="section">
+      <div class="team-info">
+        <div class="team-logo">
+          <img :src="league.logo">
+        </div>
+        <div class="team-text">
+          <h2 class="team-name">{{ league[0].name }}</h2>
+          <p class="team-subtitle">Manager: {{ league[0].manager }}</p>
+        </div>
+      </div>
+    </div>
+
+    <!-- Box container for schedule, chat, roster, standings, and statistics -->
+    <div class="box-container">
+      <NuxtLink :to='"/app/schedule"' class="box">
+        <i class="fas fa-calendar-alt"></i> <!-- Font Awesome icon for Schedule -->
+        Schedule
+      </NuxtLink>
+      <NuxtLink :to='"/app/chat"' class="box">
+        <i class="fas fa-comments"></i> <!-- Font Awesome icon for Chat -->
+        Chat
+      </NuxtLink>
+      <NuxtLink :to='"/app/schedule"' class="box">
+        <i class="fas fa-users"></i> <!-- Font Awesome icon for Roster -->
+        Roster
+      </NuxtLink>
+      <NuxtLink :to='"/app/schedule"' class="box">
+        <i class="fas fa-user-plus"></i> <!-- Font Awesome icon for Standings -->
+        Sub List
+      </NuxtLink>
+      <NuxtLink :to='"/app/schedule"' class="box">
+        <i class="fas fa-trophy"></i> <!-- Font Awesome icon for Standings -->
+        Standings
+      </NuxtLink>
+      <NuxtLink :to='"/app/schedule"' class="box">
+        <i class="fas fa-chart-bar"></i> <!-- Font Awesome icon for Statistics -->
+        Statistics
+      </NuxtLink>
+    </div>
+  </div>
+</template>
+
+<script>
+import '@fortawesome/fontawesome-free/css/all.css'; // Import Font Awesome CSS
+export default {
+  data() {
+    return {
+      league: [
+        {
+          id: 1,
+          name: 'A League',
+          logo: 'path/to/logo.png',
+          manager: 'Jacob Christensen',
+        },
+      ]
+    };
+  },
+};
+</script>
+
+<style>
+/* Add your CSS styles here */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+  background-color: #333;
+  color: #fff;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+
+.back-arrow {
+  cursor: pointer;
+}
+
+.title {
+  font-size: 1.5rem;
+}
+
+.options {
+  font-size: 1.2rem;
+}
+
+.section {
+  margin-bottom: 20px;
+  margin-top: 60px;
+}
+
+.team-box {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  position: relative; /* For positioning the lines */
+  text-decoration: none;
+  color: inherit;
+}
+
+.line {
+  width: 100%;
+  height: 1px;
+  background-color: #ccc; /* Line color */
+}
+
+.logo-name-container {
+  display: flex;
+  align-items: center;
+}
+
+.team-logo {
```

Comment:
It would be nice to have these styles in a shared stylesheet. That might be something we do in the new code base.